packages/components/tests/dummy/app/templates/components/tooltip.hbs

```diff
@@ -0,0 +1,282 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Tooltip Component"}}
+
+<Shw::Text::H1>Tooltip</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>TooltipButton</Shw::Text::H2>
+
+  <Shw::Text::H3>Content</Shw::Text::H3>
+
+  <Shw::Text::H4>On its own</Shw::Text::H4>
+  <Shw::Flex as |SF|>
+    <SF.Item @label="With a FlightIcon as content">
+      <Hds::TooltipButton @text="Here is more information" aria-label="Information">
+        <FlightIcon @name="info" />
+      </Hds::TooltipButton>
+    </SF.Item>
+
+    <SF.Item @label="With an HTML time element as content">
+      <Hds::TooltipButton @text="2023-03-30T19:18:01.684Z">
+        <time datetime="2023-03-30T19:18:01.684Z">Apr 5, 2023</time>
+      </Hds::TooltipButton>
+    </SF.Item>
+
+    <SF.Item @label="With really long content in tooltip">
+      <Hds::TooltipButton
+        aria-label="Information"
+        @text="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+      >
+        <FlightIcon @name="info" />
+      </Hds::TooltipButton>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H4>Used next to text</Shw::Text::H4>
+  <p>
+    <span class="hds-typography-display-300">Lorem ipsum dolor sit amet consectetur</span>
+    <Hds::TooltipButton @text="Here is more info" aria-label="Information"><FlightIcon
+        @name="info"
+      /></Hds::TooltipButton>
+  </p>
+
+  <p class="hds-typography-body-300">
+    Lorem ipsum dolor sit amet consectetur
+    <Hds::TooltipButton @text="Here is more info"><FlightIcon @name="info" /></Hds::TooltipButton>
+    adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+  </p>
+
+  <Shw::Text::H4>Displayed as flex (isInline = false)</Shw::Text::H4>
```

Comment:
This is under "Content" but should probably go under a "Display" section?
See how it's done in the `Dropdown` as an example:
https://hds-showcase.vercel.app/components/dropdown